README.md

```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
 ```
 
 ## Using the SDK
-Simply import the SDK and provide the API credentials provided to you. They may be an API Token or OAuth 2.0 Credentials. Examples are listed below for each feature.
+Simply import the SDK and provide the API credentials provided to you. They may 
+be an API Token or OAuth 2.0 Credentials. Examples are listed below for each 
+feature.
+
+### Customer management ###
+
+Our Customer Management service provides administrative operations to manage 
+Customer accounts and Customer User accounts for a Partner. These operations 
+allows a partner to automate administrative tasks on their customers and 
+customer user accounts. 
+
+To use this Customer Management service, use the API Token provided to you.
+
+#### Customer
+
+Customer Account Management Operations allows management of customer accounts
+
+```
+import (
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/customer"
+)
+// ...
+	sdkConfig := edgecast.NewSDKConfig()
+	sdkConfig.APIToken = "MY API TOKEN"
+	customerService, err := customer.New(sdkConfig)
+	newCustomer := customer.Customer{
+		// ...
+	}
+	addParams := customer.NewAddCustomerParams()
+	addParams.Customer = newCustomer
+	resp, err := customerService.AddCustomer(*addParams)
+	// ...
+}
+```
+
+#### Customer User
+
+Customer User Account Management Operations allows management of user accounts 
+under a (parent) customer.
+
+```
+import (
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/customer"
+)
+// ...
+	sdkConfig := edgecast.NewSDKConfig()
+	sdkConfig.APIToken = "MY API TOKEN"
+	customerService, err := customer.New(sdkConfig)
+	customerUser := customer.CustomerUser{
+		// ...
+	}
+	addParams := customer.NewAddCustomerUserParams()
+	addParams.CustomerUser = customerUser
+	// ...
+	resp, err := customerService.AddCustomerUser(*addParams)
+	// ...
+}
+```
+
+### Edge CNAME ###
+
+Our User-Friendly URL, also known as Edge CNAME, takes advantage of an Edge CNAME 
+configuration and a CNAME record to provide a friendlier alternative to a CDN URL. 
+An edge CNAME URL is specific to the platform from which it was configured.
+
+For more information, please read the [official documentation](https://docs.edgecast.com/cdn/index.html#Origin_Server_-_File_Storage/Creating_an_Alias_for_a_CDN_URL.htm)
```

Comment:
For clarity, perhaps the link text should be "official documentation for Edge CNAMEs"

Reply:
Same for the other links to the official docs

Reply:
updated